aspnetcore/blazor/host-and-deploy/webassembly.md

```diff
@@ -166,6 +166,222 @@ For more information, see the following articles:
+
+For a hosted Blazor WebAssembly app accessible via requests at the root URL (`/`), update `Program.cs` of the **`Server`** project:
+
+* Add a `using` statement to the top of the file for the <xref:Microsoft.AspNetCore.HttpOverrides?displayProperty=fullName> namespace:
+
+  ```csharp
+  using Microsoft.AspNetCore.HttpOverrides;
+  ```
+
+* Configure <xref:Microsoft.AspNetCore.Builder.ForwardedHeadersOptions> to forward the `X-Forwarded-For` and `X-Forwarded-Proto` headers:
+
+  ```csharp
+  app.UseForwardedHeaders(new ForwardedHeadersOptions
+  {
+      ForwardedHeaders = 
+          ForwardedHeaders.XForwardedFor | ForwardedHeaders.XForwardedProto
+  });
+  ```
+
+For more information, see <xref:host-and-deploy/proxy-load-balancer>.
+
+### Apps accessible at a sub-app path
+
+For a hosted Blazor WebAssembly accessible at a sub-app path (for example, `/blazor`):
+
+* In `Program.cs` of the **`Server`** project:
+
+  * Add a `using` statement to the top of the file for the <xref:Microsoft.AspNetCore.HttpOverrides?displayProperty=fullName> namespace:
+
+    ```csharp
+    using Microsoft.AspNetCore.HttpOverrides;
+    ```
+
+  * Configure <xref:Microsoft.AspNetCore.Builder.ForwardedHeadersOptions> to forward the `X-Forwarded-For` and `X-Forwarded-Proto` headers, and call <xref:Microsoft.AspNetCore.Builder.UsePathBaseExtensions.UsePathBase%2A> to add middleware that extracts the specified path base from request path and adds it to the end of the request path base:
+
+    ```csharp
+    app.UseForwardedHeaders(new ForwardedHeadersOptions
+    {
+        ForwardedHeaders = 
+            ForwardedHeaders.XForwardedFor | ForwardedHeaders.XForwardedProto
+    });
+
+    app.UsePathBase("{PATH}");
+    ```
+
+    In the preceding example, the `{PATH}` placeholder is the sub-app path. In the following example, the app's sub-app path is `/blazor`:
+
+    ```csharp
+    app.UseForwardedHeaders(new ForwardedHeadersOptions
+    {
+        ForwardedHeaders = 
+            ForwardedHeaders.XForwardedFor | ForwardedHeaders.XForwardedProto
+    });
+
+    app.UsePathBase("/blazor");
+    ```
+
+  For more information, see the following resources:
+
+  * <xref:blazor/host-and-deploy/index#app-base-path>
+  * <xref:host-and-deploy/proxy-load-balancer>
+
+* In `wwwroot/index.html` of the **`Client`** project, set the [app base path](xref:blazor/host-and-deploy/index#app-base-path) to the app's sub-app path. **A trailing slash is required.** Replace the `{PATH}` placeholder with the sub-app path in the following markup:
+
+  ```html
+  <base href="{PATH}">
+  ```
+
+  For an app that loads requests at `/blazor`, the `href` attribute is set to a value of `/blazor/`. Note that the trailing slash in the following example is required:
+
+  ```html
+  <base href="/blazor/">
+  ```
+
+* In links throughout the app, do ***not*** prefix with a forward slash. Either avoid the use of a path segment separator or use dot-slash (`./`) relative path notation:
```

Comment:
I think `NavigationManager` should be mentioned here too. It's quite common in the development of a blazor app, though it's essentially similar to `href`.

``` html
@page "/account"
@inject NavigationManager NavigationManager

<button onclick='Navi2OtherPage'>JumpToOtherPage</button>

@code{
    private void Navi2OtherPage()
    {
        NavigationManager.NavigateTo("/other"); //Incorrect
        NavigationManager.NavigateTo("other"); //Correct
        NavigationManager.NavigateTo("./other"); //Correct
    }
}
```

```